### PR TITLE
chore: update snyk cli to v1.1017.0 [ROAD-1163]

### PR DESCRIPTION
**Pull request by j-luong**
This PR should update the Snyk CLI to v1.1013.0

### CODE REVIEW

**Comment by bastiandoetsch** [comment]
I think you need to update the actual binaries, too. See https://github.com/snyk/teamcity-snyk-security-plugin/commit/814b79e2bddf8c33f3a8389eb05e89d68fdb7080

**Comment by j-luong** [comment]
> 

manually added the latest binaries from [CLI](https://github.com/snyk/cli/releases/tag/v1.1013.0) & [snyk-to-html](https://github.com/snyk/snyk-to-html/releases/tag/v2.3.6)

**Comment by bastiandoetsch** [comment]
What about the binaries for linux_aarch64 and macos-arm?

**Comment by j-luong** [comment]
I kept it to the same binaries as I wasn't sure if the plugin would support other architectures, if it's a non-issue I can add them to the PR

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: Release
+on:
+  push:
```
Comment (bastiandoetsch) [inline .github/workflows/release.yml]: workflow_dispatch:

**Review by bastiandoetsch — APPROVED** [review]

**Pull request merged**